core/src/components/toast/test/basic/toast.e2e.ts

```diff
@@ -134,3 +134,27 @@ test.describe('toast: properties', () => {
     await expect(toast).toHaveClass(/my-custom-class/);
   });
 });
+
+test.describe('toast: duration config', () => {
+  test('should have duration set to 0', async ({ page }) => {
```

Comment:
```suggestion
  test.beforeEach(({ skip }) => {
    skip.rtl();
    skip.mode('ios');
  });
  test('should have duration set to 0', async ({ page }) => {
```

One last change -- we don't need to run this for RTL/multi mode layouts.